tensorboard/program.py

```diff
@@ -432,16 +432,7 @@ def _make_data_ingester(self):
                 sys.stderr.write(msg)
                 sys.exit(1)
 
-        if flags.load_fast == "auto" and _should_use_data_server(flags.logdir):
-            if flags.logdir_spec and not flags.logdir:
-                logger.info(
-                    "Warning: --logdir_spec is not supported with --load_fast "
-                    + "behavior; falling back to multiplexer. To use the data "
-                    + "server, replace --logdir_spec with --logdir."
-                )
-                ingester = local_ingester.LocalDataIngester(flags)
-                ingester.start()
-                return ingester
```

Comment:
not quire sure what `ingester` is about but it seems missing on the changes. Is it the legacy mode mentioned in line 486?

Reply:
Yes, there used to be 2 copies of the logic to start LocalDataIngester (which is the old non-RustBoard Python logic that loads event files using EventMultiplexer).  There was one copy here (under "load_fast == auto"), if we couldn't actually use RustBoard, and then one copy as the final fallback down at the end of this function (lines 448-450 in the new version of the file).

The centralization of the logic that I'm doing in this PR consolidates those 2 copies into 1, so that the single fallback at the end of the function works for both cases.  That's because the expanded version of `_should_use_data_server()` now includes the check here (`if flags.logdir_spec and not flags.logdir`).